tutorials/SDF/car_demo.md

```diff
@@ -0,0 +1,512 @@
+            </box>
+        </geometry>
+        <!--let's add color to our link-->
+        <material>
+            <ambient>0.0 0.0 1.0 1</ambient>
+            <diffuse>0.0 0.0 1.0 1</diffuse>
+            <specular>0.0 0.0 1.0 1</specular>
+        </material>
+    </visual>
+```
+
+As the name suggests, the `<visual>` tag is responsible for how our link will look.
+We define the shape of our link inside the `<geometry>` tag as a `<box>` (cuboid) and then specify the three dimensions (in meters) of this box inside the `<size>` tag.
+Then, inside the `<material>` tag we define the material of our link.
+Here we defined the `<ambient>`, `<diffuse>` and `<specular>` colors in a set of four numbers red/green/blue/alpha each in range [0, 1].
+
+```xml
+        <collision name='collision'>
+            <geometry>
+                <box>
+                    <size>2.0 1.0 0.5</size>
+                </box>
+            </geometry>
+        </collision>
+    </link>
+</model>
+```
+
+The `<collision>` tag defines the collision properties of the link, how our link will react with other objects and the effect of the physics engine on it.
+
+**Note**: `<collision>` can be different from the visual properties, for example, simpler collision models are often used to reduce computation time.
+
+After copying all the parts above into the world file in order, run the world again:
+
+`ign gazebo car_demo.sdf`
+
+Our model should look like this:
+
+![car chassis](chassis.png)
+
+In the top right corner click on the plugins dropdown list (vertical ellipsis), choose `Transform control`, select your model and then click on the Translation tool.
+You should see three axes like this:
+
+![model_axis](model_axis.png)
+
+These are the axes of our model where red is the x-axis, green is the y-axis and blue is the z-axis.
+
+### Left wheel
+
+Let's add wheels to our robot. The following code goes after the `</link>` tag and before the `</model>` tag. All the links and joints belonging to the same model should be defined before the `</model>`.
+
+```xml
+<link name='left_wheel'>
+    <pose relative_to="chassis">-0.5 0.6 0 -1.5707 0 0</pose>
+    <inertial>
+        <mass>1</mass>
+        <inertia>
+            <ixx>0.043333</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>0.043333</iyy>
+            <iyz>0</iyz>
+            <izz>0.08</izz>
+        </inertia>
+    </inertial>
+```
+
+We defined the name of our link `left_wheel` and then defined its `<pose>` `relative_to` the `chassis` link.
+The wheel needed to be placed on the left to the back of the `chassis` so that's why we chose the values for `pose` as `-0.5 0.6 0`.
+Also, our wheel is a cylinder, but on its side.
+That's why we defined the orientation value as `-1.5707 0 0` which is a `-90` degree rotation around the x-axis (the angles are in radians).
+Then we defined the `inertial` properties of the wheel, the `mass` and the `inertia` matrix.
+
+#### Left wheel visualization
```

Comment:
This section is adding both visual and collision, so I think this title is misleading. Maybe we can just remove it?